src/codegen/irgen.cpp

```diff
@@ -785,14 +859,13 @@ static void emitBBs(IRGenState* irstate, TypeAnalysis* types, const OSREntryDesc
                 ASSERT(it->second.first == v->getType(), "");
                 llvm_phi->addIncoming(v->getValue(), osr_unbox_block_end);
             }
-
-            InternedString is_defined_name = getIsDefinedName(it->first, source->getInternedStrings());
         }
         for (auto t : phi_args) {
             std::get<0>(t)->addIncoming(std::get<1>(t), std::get<2>(t));
         }
     }
 
+    // TODO(rntz): what are we doing in this for-loop?
```

Comment:
This is just trying to clean up some of the codegen memory -- `decvrefNodrop()` just drops a "vref" which will eventually delete the object once it goes to zero.  The memory management can be improved a lot but hopefully that's not what you're trying to get at with this question :)